content/sigs/docs/gsod/2020/projects/document-jenkins-on-kubernetes.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: gsodproject
+title: "Document Jenkins on Kubernetes"
+goal: "Create a new Kubernetes Volume which would describe the concepts, techniques, and choices for Kubernetes users running Jenkins."
+category: Documentation
+year: 2020
+sig: jenkins-documentation
+status: "Active"
+student: zaycodes
+mentors:
+- "oleg_nenashev"
+- "MarkEWaite"
+- "kwhetstone"
+- "markyjackson-taulia"
+links:
+  gitter: jenkinsci/docs
+  draft: https://docs.google.com/document/d/1zTEKtOp2i1K2fw5RQ_a_KVOB2z0gz9987NYzTnIS6G8/edit?usp=sharing
+  idea: /sigs/docs/#jenkins-on-kubernetes
+tags:
+- gsod2020
+- kubernetes
+- documentation
+- guide
+---
+
+=== Abstract
+
+Kubernetes is a platform-agnostic container orchestration tool created by Google and heavily supported by the open-source community as a project of the Cloud Native Computing Foundation. It allows you to use container instances and manage them for scaling and fault tolerance. It also handles a wide range of management activities that would otherwise require separate solutions or custom code, including request routing, container discovery, health checks, and rolling updates.
```

Comment:
I think another thing to note here is we normally put each sentence on a different line.  This makes it easy to suggest changes and see changes in the project.